source/documentation/deploying_services/index.md

```diff
@@ -157,6 +189,65 @@ where `SERVICE_NAME` is a unique descriptive name for this instance of the servi
 
 Run `cf conduit --help` for more options, and refer to the [Conduit readme file](https://github.com/alphagov/paas-cf-conduit/blob/master/README.md) [external link] for more information on how to use the plugin.
 
+### Import and export bulk data to and from a PostgreSQL database
+
+You must have the PostgreSQL command line tools installed on your local machine for this task.
+
+#### Non-PaaS to PaaS
+
+To move data from a non-PaaS hosted PostgreSQL database to a PaaS-hosted PostgreSQL database:
+
+1. Run the following code in the command line to export data from the non-PaaS hosted database to an SQL data file:
+
+    ```
+    pg_dump --host HOST_NAME --file DATA_FILE_NAME DATABASE_NAME
+    ```
+
+    where:
+    - `HOST_NAME` is the name of your host
+    - `DATA_FILE_NAME` is the SQL data file
+    - `DATABASE_NAME` is the name of the origin non-PaaS hosted database
+
+2. Ensure that you have:
+    - [logged into Cloud Foundry](/#setting-up-the-command-line)
+    - [created the new PaaS-hosted PostgreSQL database](/#set-up-a-postgresql-service)
+    - [targeted the space](/#setting-a-target) where your new database is located
+
+3. Use the [Conduit plugin](/#connect-to-a-postgresql-service-from-your-local-machine) to import the data file into the PaaS-hosted database by running:
+
+    ```
+    cf conduit SERVICE_NAME -- psql < DATA_FILE_NAME
+    ```
+
+    where `SERVICE_NAME` is a unique descriptive name for this instance of the service, and `DATA_FILE_NAME` is the SQL file created in the previous step.
+
+If your source database uses PostgresSQL extensions, you may see some extension errors. Contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) if you have any questions.
+
+#### PaaS to PaaS
```

Comment:
It should actually be possible to do this in one line between different orgs/spaces:

```
cf conduit -o ORG1 -s SPACE1 -p 7000 SOURCE_SERVICE -- pg_dump | cf conduit -o ORG2 -s SPACE2 -p 7001 DEST_SERVICE -- psql
```

which is useful if your database is extremely large becuase it will not require storing on disk, and can be performing the upload at the same time as the download.... however it's a bit less clear what is going on so I'm not sure if it's worth mentioning... I'll leave it up to you

Reply:
Agreed not necessary